index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <main id="main">
+      <h1 id="title">Dr. Norman Borlaug</h1>
+      <p>The man who saved a billion lives</p>
+      <div id="img-div">
+        <div id="div">
+        <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+        alt="Dr. Norman Borlaug seen standing in Mexican wheat field with a group of biologists">
+        <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p>
+        </div>
+        <div id="body">
+         <h2 id="tribute-info">Here's a time line of Dr. Borlaug's life:</h2>
+         <ul>
+            <li>1914 - Born in Cresco, Iowa</li>
+            <li>1933 - Leaves his family's farm to attend the University of Minnesota, thanks to a Depression era program known as the "National Youth Administration"</li>
+            <li>1935 - Has to stop school and save up more money. Works in the Civilian Conservation Corps, helping starving Americans. "I saw how food changed them", he said. "All of this left scars on me."</li>
+            <li>1937 - Finishes university and takes a job in the US Forestry Service</li>
+            <li>1938 - Marries wife of 69 years Margret Gibson. Gets laid off due to budget cuts. Inspired by Elvin Charles Stakman, he returns to school study under Stakman, who teaches him about breeding pest-resistent plants.</li>
+            <li>1941 - Tries to enroll in the military after the Pearl Harbor attack, but is rejected. Instead, the military asked his lab to work on waterproof glue, DDT to control malaria, disinfectants, and other applied science.</li>
+            <li>1942 - Receives a Ph.D. in Genetics and Plant Pathology</li>
+            <li>2009 - dies at the age of 95.</li>
+         </ul>
+         </div>
+         <p>If you have time, you should read more about this incredible human being on his <a id="tribute-link"  href="https://en.wikipedia.org/wiki/Norman_Borlaug">Wikipedia entry.</a></p>
```

Comment:
I suggest this should be outside the #main. It will be better of with it own styling.

Reply:
- I'd like to kindly suggest that you state reasons WHY this changes needs to be made. For example,  you can rephrase this to something like ` I suggest this should be outside the #main. It will be better of with it own styling because outside the main has .  . . . . . . . ` .
- Feep free to state a valid reason why you require a change. 😊